### PR TITLE
chore(deps): bump resty-openssl to 0.8.20

### PR DESCRIPTION
**Pull request by chronolaw**
### Summary

bump resty-openssl to 0.8.20
add missing change log entry for resty-timer-ng 0.2.4

### Checklist

- [ ] The Pull Request has tests
- [x] There's an entry in the CHANGELOG
- [ ] There is a user-facing docs PR against https://github.com/Kong/docs.konghq.com - PUT DOCS PR HERE

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -130,6 +130,7 @@
   [#10137](https://github.com/Kong/kong/pull/10413)
 - Bumped lua-resty-openssl from 0.8.17 to 0.8.18
```
Comment (fffonion) [inline CHANGELOG.md:131]:
```suggestion
- Bumped lua-resty-openssl from 0.8.17 to 0.8.20
```

Comment (chronolaw) [reply CHANGELOG.md]: many thanks.

**Review by bungle — APPROVED** [review]

**Comment by bungle** [comment]
@chronolaw there seems to be 4 commits, can you rebase and make it 2 commits, one for openssl bump and one for timer-ng docs update.

**Comment by chronolaw** [comment]
> @chronolaw似乎有 4 次提交，您能否重新设置基准并使其成为 2 次提交，一次用于 openssl bump，一次用于 timer-ng 文档更新。

OK, I will do it later.

**Comment by bungle** [comment]
The commits are not following proper prefixes, so hold on merge.

**Comment by chronolaw** [comment]
> The commits are not following proper prefixes, so hold on merge.

Done. Thank you for pointing this.

**Review by bungle — APPROVED** [review]

**Pull request merged**